docs/source/getting-started/tabular-data/index.rst

```diff
@@ -155,16 +150,6 @@ Context Parameters
   The privacy loss is upper-bounded by privacy parameters; in this case epsilon (ε).
 * ``split_evenly_over``: This is the number of queries you want to distribute your privacy loss over. 
   Configure this parameter appropriately according to how many queries you would like to release. 
-* ``margins``: Margins capture public information about groupings of your dataset.
```

Comment:
I dropped this because I felt that it would be better to introduce these concepts as needed for statistics.